src/utils/event_utils.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from 'lodash';
+import * as Web3 from 'web3';
+import {EventCallback, ContractEventArg, ContractEvent, ContractEventObj, ContractEventEmitter} from '../types';
+import * as BigNumber from 'bignumber.js';
+import promisify = require('es6-promisify');
+
+export const eventUtils = {
+    /**
+     * Wraps eventCallback function so that all the BigNumber arguments are wrapped in a newer version of BigNumber.
+     * @param eventCallback     Event callback function to be wrapped
+     * @return Wrapped event callback function
+     */
+    getBigNumberWrappingEventCallback(eventCallback: EventCallback): EventCallback {
```

Comment:
This should have an underscore in-front of its name since it is only called from within this module.